caput/pipeline.py

```diff
@@ -312,6 +312,8 @@
 from future.builtins import *  # noqa  pylint: disable=W0401, W0614
 from future.builtins.disabled import *  # noqa  pylint: disable=W0401, W0614
 
+from six import string_types
```

Comment:
I think here you should use `from past.builtins import basestring` and use `basestring` where you've used `string_types`. That's what we're doing elsewhere in the code.